src/PhpSpreadsheet/Reader/Xls.php

```diff
@@ -417,6 +417,9 @@ class Xls extends BaseReader
      */
     private $baseCell;
 
+    /** @var bool */
+    private $activeSheetSet = false;
```

Comment:
Please refrain from introducing non-native typing. Now that we require PHP 8.0 we should be able to natively type almost everything, especially properties.

Reply:
Will do.